render.go

```diff
@@ -160,8 +164,8 @@ func (r *Render) renderCompletion(completions *CompletionManager, cursorPos int)
 		r.out.CursorForward(x + width - int(r.col))
 	}
 
-	r.out.CursorUp(windowHeight)
 	r.out.SetColor(DefaultColor, DefaultColor, false)
+	return int(r.col) * windowHeight // the number of characters to go up
```

Comment:
Could you explain this part a bit? Why do we need to multiply the height by the number of columns now?

Reply:
Basically, I wanted render functions to be a bit more standardized and return the amount of characters to go back and that's why I transformed `r.out.CursorUp(windowHeight)` into `int(r.col) * windowHeight`

Reply:
We resolved this discussion offline